sdk/appconfiguration/azure-data-appconfiguration/swagger/README.md

```diff
@@ -132,3 +133,68 @@ directive:
         }
       ];
 ```
+
+### Modify SnapshotField enums
+```yaml
+directive:
+  - from: swagger-document
+    where: $.parameters.SnapshotFields
+    transform: >
+      $.items["x-ms-enum"].values = [
+        {
+          "value": "name",
+          "name": "name",
+          "description": "Populates the snapshot `name` from the service."
```

Comment:
Let's not use `'` instead of `

`"description": "Populates the snapshot 'name' from the service"`